src/camvidlog2/bioclip/cli.py

```
@@ -0,0 +1,150 @@
+from camvidlog2.bioclip.queries import (
+    calculate_distances,
+    calculate_results,
+    load_embedding_dataframe,
+)
+from camvidlog2.vid import FrameError, generate_frames_cv2, get_frame_by_no, save
+
+app = typer.Typer()
+
+
+@app.command()
+def load(
```

Ruff (0.11.9)

Missing return type annotation for public function `load`

Add return type annotation: `None`

(ANN201)